README.md

```diff
@@ -32,6 +32,7 @@ require 'vendor/deployphp/recipes/recipes/cachetool.php';
 | ------    | ----                                                                      | -----
 | cachetool | [read](http://github.com/deployphp/recipes/blob/master/docs/cachetool.md) | `require 'vendor/deployphp/recipes/recipes/cachetool.php';`
 | newrelic  | [read](http://github.com/deployphp/recipes/blob/master/docs/newrelic.md)  | `require 'vendor/deployphp/recipes/recipes/newrelic.php';`
+| local     | [read](http://github.com/deployphp/recipes/blob/master/docs/local.md)     | `require 'vendor/deployphp/recipes/recipes/local.php';`
```

Comment:
Please move this after `cachetool` and before `newrelic` :) 
lets keep it in alphabetical order